src/codewhisperer/util/telemetryHelper.ts

```diff
@@ -202,19 +200,30 @@ export class TelemetryHelper {
             telemetry.codewhisperer_userDecision.emit(event)
             events.push(event)
         })
+
+        //aggregate suggestion references count
+        this.getAggregatedSuggestionReferenceCount(events)
+
         // aggregate user decision events at requestId level
         const aggregatedEvent = this.aggregateUserDecisionByRequest(events, requestId, sessionId)
         if (aggregatedEvent) {
             this.sessionDecisions.push(aggregatedEvent)
         }
 
+        let acceptedRecommendationContent
+        if (acceptIndex != -1 && recommendations[acceptIndex] != undefined) {
+            acceptedRecommendationContent = recommendations[acceptIndex].content
+        } else {
+            acceptedRecommendationContent = ''
```

Comment:
using a ternary would allow this to be `const` and is generally more readable and less verbose